modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "profiles": {
+    "Microsoft.Azure.IIoT.Modules.OpcUa.Publisher": {
+      "commandName": "Project",
+      "commandLineArgs": "--ec=HostName=iothub-v3m2qo.azure-devices.net;DeviceId=crpogace01;ModuleId=publisher;SharedAccessKey=ibnTVaat5jDBUImD0B6AR6en2q/Z2hqh+NDE0ehpnBY="
```

Comment:
@cristipogacean, please refresh access keys for this device as this one is already compromised. This also looks like a change done for a local debugging. So please revert changes to this file.

Reply:
reverted